aikido/src/util/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ aikido_add_library("${PROJECT_NAME}_util"
   VanDerCorput.cpp
   StepSequence.cpp
   RNG.cpp
+  stream.cpp
```

Comment:
I don't know how I feel about other cpp source files being camel case because they define classes and this one being all lowercase.  Is there a convention for this?